### DISCUSSION TITLE
[17] Dinder - Retrospektive

### DISCUSSION THREAD

**Discussion by Manuel-F-04** [dhbw-ka-tinf22b5-dinder/Dinder#30] [Projektblog]
## Retrospektive
Das Board befindet sich [hier](https://github.com/dhbw-ka-tinf22b5-dinder/Dinder-SRS/blob/main/Retrospektive.pdf)
### Zusammenfassung
- Ausbau der Kommunikation -> feste verbindliche Meetingzeiten setzen 
- genauere Betrachtung des Tech-Stacks am Anfang des Projekts -> Was ist bekannt? Wie viel Aufwand steckt dahinter?

## Projektfortschritt
- Abschließung der Implementierung (Swiped Advertisements, Published Advertisements)
- kleinere Korrekturen am Backend, sowie am Frontend
- Erstellung neuer Test -> ~64% Coverage
- Fertigstellung der Projektdokumentation

**Comment by ghost** [comment]
Spannende Ansätze die ihr in eurer Retrospektive benennt. Man sieht auch, dass ihr direkt schlüssige Maßnahmen aus ihnen ziehen konntet.

**Comment by DossohMael** [comment]
Die von euch gewählten Bewertungskriterien gefallen mir sehr, sie wären für den zukünftigen Verlauf des Projekts von großem Nutzen.
